brainvisa/hierarchies/brainvisa-3.2.0/volbrain.py

```diff
@@ -64,6 +63,33 @@ def volbrain_space(space):
             ),
             "report_<subject>",
             SetType("Analysis Report"),
+            "metadata_<subject>",
+            SetType('Metadata Execution')
         ),
     ),
 )
+
+insert('{center}/{subject}',
+    'vol2Brain', SetWeakAttr('modality', 'vol2Brain'), SetContent(
+        '{acquisition}', SetType('Acquisition'),
+        SetDefaultAttributeValue('acquisition', default_acquisition), SetContent(
+            'matrix_affine_native_to_mni_<subject>', SetType('Transformation'),
+            'mni_t1_<subject>', SetType('T1 MRI Denoised and Bias Corrected'), SetWeakAttr('space', 'mni'),
+            'mni_mask_<subject>', SetType('Intracranial mask'), SetWeakAttr('space', 'mni'),
+            'mni_tissues_<subject>', SetType('Intracranial labels'), SetWeakAttr('space', 'mni'),
+            'mni_macrostructures_<subject>', SetType('Split Brain Mask'), SetWeakAttr('space', 'mni'),
+            'mni_lobes_<subject>', SetType('Brain Lobes'), SetWeakAttr('space', 'mni'),
+            'mni_structures_<subject>', SetType('Subcortical labels'), SetWeakAttr('space', 'mni'),
+            'mni_thickness_<subject>', SetType('Cortical Thickness map'), SetWeakAttr('space', 'mni'),
```

Comment:
We could use your `assemblynet_space` function if we add the `thickness` file in it, with a comment that it's for vol2brain and not assemblyNet. What do you think ?

Reply:
In fact we could use the `assemblynet_space` function as it is and just add `[mni/native]_thickness_`.